spyder/plugins/editor/plugin.py

```diff
@@ -327,10 +327,10 @@ def report_open_file(self, options):
                 codeeditor.register_completion_capabilities(
                     self.completion_capabilities[language.lower()])
                 codeeditor.start_completion_services()
-            elif self.main.completions.is_fallback_only(language.lower()):
-                # This is required to use fallback completions for files
-                # without a language server.
-                codeeditor.start_completion_services()
+            # elif self.main.completions.is_fallback_only(language.lower()):
+            # This is required to use fallback completions for files
+            # without a language server.
+            codeeditor.start_completion_services()

```

Comment:
I think this change is incorrect. We can't start the completion services unconditionally or things will break.

I mean, it took me a long time to fix this so I'd like to hear a good reason on why your changed this.